www/pages/my-buses/mybuses.html

```diff
@@ -49,7 +49,7 @@
       <ion-item class="item item-text-wrap" ng-repeat="trip in trips track by $index" ng-click="openTrip($index)">
         <ion-option-button class="button-assertive icon ion-trash-a" ng-click="removeTrip($index)"></ion-option-button>
         <div class="item-icon-left" aria-label="{{trip.name}}">
-          <i class="icon ion-map" style="margin-right: 10px"></i>
+          <i class="icon ion-map" style="margin-right: 10px" aria-label="schedule"></i>
```

Comment:
Even though the page is called `Schedule`, an entry in this list wouldn't be called a "schedule," right?  I'd say either label it "schedule _icon_" or "trip"